articles/email/providers.md

```diff
@@ -1,60 +1,119 @@
-# Using your own SMTP Provider for sending emails
+# Use your own SMTP Email Provider
```

Comment:
One thing I forgot to mention is that we are no longer recommending (or supporting) the use of Auth0's email provider for production use - You now must use your own custom provide in production. So Lets add something that nots that the auth0 provide is for testing, etc. and you should use your preferred provider before you switch to production.

I would also recommend we add links showing where and how users can sign up to each of our supported services. We dont need to write those docs, the other providers should have them.